.github/workflows/weekly-pulumi-update.yml

```diff
@@ -42,13 +44,12 @@ jobs:
 
         git checkout -b update-pulumi/${{ github.run_id }}-${{ github.run_number }}
 
-        go get github.com/pulumi/pulumi/pkg/v3
+        PULUMI_VERSION=$(pulumi version)
```

Comment:
A bit of an approximation here.. Assuming we are going to compute the latest version of SDK and PKG by computing the latest version of Pulumi CLI. Seems cleanest for now as they do in practice update in sync. Can be refined though to find the latest version of Pulumi Go SDK specifically.